### PR TITLE
Feature/edit information

### PR DESCRIPTION
**Pull request by StrongCurrent**
Now entries and the spot name are also editable. I decided on a solution with buttons because it is simply better and easier for the user to understand than just clicking somewhere.

### CODE REVIEW

File: components/SpotInfo/index.js [hunk 1]
```diff
@@ -109,10 +230,36 @@ export default function SpotInfo({ spotId }) {
           <EntryList>
```
Comment (Roland-Hufnagel) [inline components/SpotInfo/index.js:230]: Your ``<EntryList>`` is a ``ul``. That's fine. But the direct children are ``<div>``s! They have to be ``<li>``s.
Your ``<li>`` is two areas below.

**Review by Roland-Hufnagel — CHANGES_REQUESTED** [review]
Hi Martin. As I said, you should try to keep things simple. I gave you an example. I left a comment about your list, you should fix this. And make sure that the 'Delete Spot' Button is always visible.

**Review by Roland-Hufnagel — CHANGES_REQUESTED** [review]
Hi Martin, you fixed the button problem but not the list problem. Since this is semantically a typical list, you have to use ``<ul>``and ``<li>``.

The rest seems to work but as I mentioned I did not dive deep in because it's that much complex.

Two more things I want you to mention: An Input-Element should always live in a form (and the handleSubmit makes it work) and you use 'Controlled Inputs' but I think that's not necessary in your case.

File: components/SpotInfo/index.js [hunk 2]
```diff
@@ -108,39 +124,14 @@ export default function SpotInfo({ spotId }) {
         {spot.informations && spot.informations.length > 0 ? (
           <EntryList>
             {spot.informations.map((entry) => (
-              <EntryCard key={entry._id}>
-                <EntryTextarea>{entry.info}</EntryTextarea>
-                <EntryDeleteButton onClick={() => handleDeleteEntry(entry._id)}>
-                  <StyledIcon as={PiTrash} size={25} />
-                </EntryDeleteButton>
-              </EntryCard>
+              <EditDeleteInfoForm key={entry._id} entry={entry} spotId={spotId} />
```
Comment (Roland-Hufnagel) [inline components/SpotInfo/index.js]: Yes! That's it 👍
But why do you send the 'spotId' as a prop? I think, this information is already included in the 'entry'-prop

**Review by Roland-Hufnagel — APPROVED** [review]
Hi Martin. I left one more comment you can look at. But all in all it looks a bit cleaner now. Even if it is still more complex as it has to be 🙂. But it works. Please make the comments in the spot to be a list and then you're on the right way 🚀

File: components/SpotInfo/index.js (hunk 2 above)
Comment (StrongCurrent) [reply components/SpotInfo/index.js]: I'll look at it again but for now I want to get to my modal. THX

**Pull request merged**